src/applications/personalization/profile/components/ProfileWrapper.jsx

```diff
@@ -84,15 +74,7 @@ const ProfileWrapper = ({
       <div data-testid="breadcrumbs">
         <Breadcrumbs className="vads-u-padding-x--1 vads-u-padding-y--1p5 medium-screen:vads-u-padding-y--0">
           <a href="/">Home</a>
-
-          {showLOA1BreadCrumb && <Link to="/">Profile - Account security</Link>}
-
-          {!showLOA1BreadCrumb &&
-            !onPersonalInformationMobile && <Link to="/">Profile</Link>}
-
-          {!showLOA1BreadCrumb && (
-            <a href={activeLocation}>{activeRouteName}</a>
-          )}
+          <a href={activeLocation}>{`Profile: ${activeRouteName}`}</a>
```

Comment:
The breadcrumbs are _waaaaaay_ simpler now. It's always just the Home link and the current section.

Reply:
Yeah.. the previously existing logic always felt a bit hacky - happy that the requirements have been simplified 👍